### ISSUE TITLE
pscp could not parse when userid contains dash(-) 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jimradford/superputty#440]
```
What steps will reproduce the problem?
1.when userid contains dash(-)
2.click File Transfer
3.Remote Folder is blank

What is the expected output? What do you see instead?
support even if userid contains dash

What version of the product are you using? On what operating system?
1.4.0.5

Please provide any additional information below.

<Log Viewer>
PscpClient - Could not parse directory listing entry
```

Original issue reported on code.google.com by `hiky...@gmail.com` on 15 Jul 2014 at 4:36

**Comment by jimradford** [comment]
Related to issue #533 Will look into this.

**Comment by jimradford** [comment]
Fixed and committed to master. Will be included in 1.0.4.7 release

**Comment by csabasolyom** [comment]
Issue with "-" in username persists in v.1.4.0.8
